tests/e2e/utils/auto-rds-s3-setup.py

```diff
@@ -42,12 +71,16 @@ def verify_oidc_provider_prerequisite():
     if is_oidc_provider_present():
         print("OIDC provider found")
     else:
-        raise Exception(f"Prerequisite not met : No OIDC provider found for cluster '{CLUSTER_NAME}'!")
+        raise Exception(
+            f"Prerequisite not met : No OIDC provider found for cluster '{CLUSTER_NAME}'!"
+        )
 
 
 def is_oidc_provider_present() -> bool:
```

Comment:
lets remove this since we have it in pre-requisites, this is also relying on `alpha.eksctl.io/cluster-name`